target/scripts/startup/setup-stack.sh

```diff
@@ -259,7 +259,7 @@ function _setup_dovecot
   fi
 
   chown docker:docker -R /usr/lib/dovecot/sieve*
-  chmod 550 -R /usr/lib/dovecot/sieve*
+  chmod 755 -R /usr/lib/dovecot/sieve-pipe /usr/lib/dovecot/sieve-filter /usr/lib/dovecot/sieve-global
```

Comment:
Since this is covered by the `Dockerfile` already, is there any need for this line here too? Or is it due to changes since at runtime that we or user adds?

I guess there's no harm in doing so, I don't mind either way. I had a quick search for `dovecot sieve chmod` on the web and nothing seemed to stand out, it's unclear why root was being made read-only here by original author.

Reply:
> Since this is covered by the Dockerfile already, is there any need for this line here too? Or is it due to changes since at runtime that we or user adds?

Good point. Maybe just removing the `chmod` line is enough.

@17Halbe Can you explain, what your intention was?

Reply:
> I don't mind the tests, I'm just not sure how useful they are at present.

I've added them to prevent future regression made by any PR.

Reply:
I understand why you added them, but in this case a regression was never introduced, the behaviour is the same since it was contributed. This wasn't different before and missed in a review by maintainers due to passing tests (_other than the original PR itself where it wasn't questioned/noticed_).

I don't think we have permissions check tests where we do this elsewhere as well? It's fine if you think there is value in it still though.

---

I wouldn't expect @17Halbe to recall such a detail from almost 4 years ago :sweat_smile: but maybe! :crossed_fingers: